### DISCUSSION TITLE
v2.0.0：同步更新《钻石／珍珠》补丁以及其他重要更新

### DISCUSSION THREAD

**Discussion by Xzonn** [Xzonn/PokemonChineseTranslationRevise#24] [Announcements]
# 《宝可梦》第四世代汉化修正补丁
## 基本说明
本项目对《宝可梦》第四世代日文版进行汉化，对原汉化组使用的旧译名进行了修正，并且统一了字符码表，以便汉化版本之间连接交换及对战。

详细说明请访问 <https://xzonn.top/PokemonChineseTranslationRevise/>。

本补丁采用“[署名-非商业性使用-相同方式共享 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/deed.zh)”进行许可。

**由于本次更新内容较多，请仔细阅读下面的说明。**

如需转载，请保留此说明。

## 版本发布说明
- 2024-02-04（2.0.0）：
  - **应用补丁所需要的原始 ROM 由原汉化版本修改为日本版**。请注意，《钻石／珍珠》需要 Rev 6 版本，以其他版本作为原始 ROM 会导致生成的 ROM 无法运行。补丁应用工具会自动计算原始 ROM 的 MD5 校验码，如果没有提示“原始 ROM 的 MD5 校验失败”，则说明原始 ROM 的版本正确。
  - **同步更新《钻石／珍珠》补丁**，并修改《钻石／珍珠》的图标。（感谢 **[ppllouf](https://github.com/ppllouf)**）
  - 采用了更细致的匹配算法，导入了更多与《晶灿钻石／明亮珍珠》等官方中文化游戏匹配的文本。
  - 版本号修改为删除存档界面显示，可在标题界面按`↑ + SELECT + B`查看。
  - 修复招式文本与实际效果不一致的问题。此问题自从导入官方招式文本后就一直存在，此前未能引起注意。
  - 修复《心金》图鉴文本实际为《魂银》图鉴文本的问题。（感谢 **Konyaka** 反馈）
  - 修复《钻石／珍珠／白金》无法开启神秘礼物的问题。神秘礼物的开启密语为“大家 快乐 Wi-Fi 连接”。（感谢 **[兔隐](https://tieba.baidu.com/home/main?id=tb.1.b078b4c8.5EUyhmk8zkrkK__di08swQ)** 等人反馈，[链接](https://tieba.baidu.com/p/7213514184)）
  - 修复《白金》标题动画显示错误的问题。（感谢 **[ppllouf](https://github.com/ppllouf)**）
  - 修复几处小图标未更新至官译的问题。
  - 修改《钻石／珍珠／白金》简单会话的词语排序。**注意，这一修改会导致日版《钻石／珍珠》壁纸密语不适用于汉化版**。请参考[“密语计算器”](https://xzonn.top/PokemonChineseTranslationRevise/Aikotoba.html)页面的说明。
  - 修复了多处翻译不准确、不恰当的地方。（非常感谢 **[爱儿aiko](https://space.bilibili.com/101749351)** 细心反馈并提出了很多修改建议）

## 使用方式
请自行获取游戏日文版 ROM，然后下载 [修正补丁](https://github.com/Xzonn/PokemonChineseTranslationRevise/releases/download/v2.0.0/PokemonChineseTranslationRevise.v2.0.0.zip) 并解压，按照压缩包中提供的 [补丁应用工具](https://github.com/Xzonn/NitroPatcher/releases/latest/) 的说明进行操作。

日文版 ROM 的 MD5 校验码：

- [《钻石》](https://datomatic.no-intro.org/index.php?page=show_record&s=28&n=6641)：`c1d66b8d4fbdbfa57ff4868970fe19d2`
- [《珍珠》](https://datomatic.no-intro.org/index.php?page=show_record&s=28&n=4929)：`751d4a0524e4ef38b153ccfb5d21271f`
- [《白金》](https://datomatic.no-intro.org/index.php?page=show_record&s=28&n=2641)：`8271f64f5c7fb299adf937a8be6d8c88`
- [《心金》](https://datomatic.no-intro.org/index.php?page=show_record&s=28&n=4168)：`e3f7933aee8cc2694629293f16c1c0a8`
- [《魂银》](https://datomatic.no-intro.org/index.php?page=show_record&s=28&n=4169)：`a1211b2d44e551197712177e3d50b491`

应用补丁后，ROM 的校验码应为（2.0.0 版本）：

- 《钻石》：
  - CRC32：`e6b03269`
  - MD5：`f37cd0f5272bfa69eb2ff58e45dfaf12`
  - SHA-1：`e1a63da0b2dc983807895340474b5af95b9d7f10`
- 《珍珠》：
  - CRC32：`eccdfe2e`
  - MD5：`af267e681450da50211e9a248880fb5d`
  - SHA-1：`3338baa8c33a3ca757deebc0a3e8d7a15fae6a1e`
- 《白金》：
  - CRC32：`d50b0804`
  - MD5：`4133c9053a5fe47b815184e0e725671e`
  - SHA-1：`e13d82776b3ac5aac6c338017e94d7518bf6cf36`
- 《心金》：
  - CRC32：`c898ab5a`
  - MD5：`07fc5de0dd108469ca47734c004f3059`
  - SHA-1：`75b1f137212bb5d4e45256ddb329e48803b61965`
- 《魂银》：
  - CRC32：`b07a4de4`
  - MD5：`82c48bcb3bc987b577f341e60511b93b`
  - SHA-1：`4725b794414477329a30d37bcee48b50682e203c`

## 授权协议
本项目使用 **[CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/legalcode)** 协议授权。若您想基于本项目进行二次创作，请遵守协议内容。这些内容主要包括：

- **署名**：您必须标明本项目的作者（Xzonn），并在您的作品中提供指向本项目的链接。
- **非商业性使用**：您不得将本项目用于商业目的。
- **相同方式共享**：若您基于本项目进行二次创作，您必须以相同的协议授权您的作品。

请阅读本项目的 **[完整授权协议](LICENSE)** 以了解更多信息。

<hr /><em>This discussion was created from the release <a href='https://github.com/Xzonn/PokemonChineseTranslationRevise/releases/tag/v2.0.0'>v2.0.0：同步更新《钻石／珍珠》补丁以及其他重要更新</a>.</em>

**Comment by clavical** [comment]
珍珠钻石金手指用不了。
